.gitignore

```diff
@@ -117,3 +117,7 @@ spark-warehouse/
 
 # For Node.js
 node_modules
+
+# For Antlr
+sql/catalyst/gen/
+*.tokens
```

Comment:
All the entries we need to add are:
```
sql/catalyst/gen/
sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/gen/
```
right?

Reply:
In my case, `sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/gen/` is never generated but `sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseLexer.tokens` is instead.